### PR TITLE
[T-221]: CMS page rendering

### PR DESCRIPTION
**Pull request by michalpulpan**
Implemented rendering of CMS pages on storefront.
There's a fallback route `pages/[slug].tsx` which will catching everything unmatched. It tries to fetch data from backend (based on `slug` and `locale` in the request). 
If nothing returned, we get `404.tsx`

Nothing too fancy, but it gets job done fairly well.

Hope you like it 😊 

https://github.com/ecoseller/ecoseller/assets/34132752/52d90ada-15d5-4766-a118-a639b78c24ad

### CODE REVIEW

File: src/cache/dump.rdb
Comment (VL-CZ) [inline src/cache/dump.rdb:1]: This file got somehow mysteriously back into our repo 😄

**Comment by VL-CZ** [comment]
Side note:
After I saw your video, I noticed that after changing language, the mega menu language stays the same (which makes sense, since it's not re-rendered after locale change). Maybe we should do full reload of page, or just redirect to home?
What do you think?

**Comment by michalpulpan** [comment]
> Side note:
> After I saw your video, I noticed that after changing language, the mega menu language stays the same (which makes sense, since it's not re-rendered after locale change). Maybe we should do full reload of page, or just redirect to home?
> What do you think?

Yeah, let's do a reload. I wouldn't do redirect to homepage 😊

File: src/cache/dump.rdb
Comment (michalpulpan) [reply src/cache/dump.rdb:1]: Hopefully removed it.
I don't know why it's still listed in the `File changes`, but it's not present in the file browser of this branch.

Snímek obrazovky 2023-06-28 v 9 32 35

**Review by VL-CZ — APPROVED** [review]

**Pull request merged**